gov/proposals.py

```diff
@@ -0,0 +1,106 @@
+import textwrap
+import asyncio
+import subprocess
+from constants import GOVERNANCE_BUDGET_CHANNEL_ID, GOVERNANCE_CHANNEL_ID
+from config.config import current_governance_id, current_budget_id, update_id_values
+
+proposals = []
+
+ongoing_votes = {}
+
+def get_next_governance_id():
+    global current_governance_id
+    current_governance_id += 1
+    return current_governance_id
+
+def get_next_budget_id():
+    global current_budget_id
+    current_budget_id += 1
+    return current_budget_id
```

Comment:
What are these doing and why are the globalizing? Don't be afraid to delete my stuff

Reply:
> What are these doing and why are the globalizing? Don't be afraid to delete my stuff

Using this to increment the budget / governance IDs when a draft is published.

I was initially using them as a global var because it seemed like that the variables were being declared locally. I have changed the import statement and am no longer globalizing.

```
import config.config as cfg

def get_next_governance_id():
    cfg.current_governance_id += 1
    return cfg.current_governance_id

def get_next_budget_id():
    cfg.current_budget_id += 1
    return cfg.current_budget_id

```